README.md

```diff
@@ -107,6 +107,42 @@ RSpec.describe "Posts Controller", type: :request do
       response 200, description: "success"
     end
   end
+
+  # Path Update Object
+  path '/posts/{post_id}' do
+    # Parameter Object for header
+    parameter 'Content-Type', {in: :header, type: :string}
+    let(:'Content-Type') { 'application/json' }
+	# ^same can be used for *Authorization* header
```

Comment:
Oh, so I don't think the docs make this clear but if you specify a single consumes type:
```rb
consumes 'application/json'
```
It'll set the `Content-Type` header to that automatically. If you specify multiple types it will select the first one.

I think that ought to be added to the docs for that method.